Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs

```diff
@@ -5,63 +5,286 @@
 using Microsoft.AppCenter.Utils.Files;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using SQLitePCL;
 using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.AppCenter.Test.Storage
 {
     [TestClass]
     public class StorageAdapterTest
     {
+        private StorageAdapter _adapter;
+
+        // Const for storage data.
+        private const string StorageTestChannelName = "storageTestChannelName";
+        private const string TableName = "LogEntry";
+        private const string ColumnChannelName = "Channel";
+        private const string ColumnLogName = "Log";
+        private const string ColumnIdName = "Id";
+        private const string DatabasePath = "databaseAtRoot.db";
+
+        [TestInitialize]
+        public void InitializeStorageTest()
+        {
+            _adapter = new StorageAdapter();
+        }
+
         [TestMethod]
         public void InitializeStorageCreatesStorageDirectory()
         {
-            var adapter = new StorageAdapter("path/to/database/file.db");
+            var adapter = new StorageAdapter();
+            Microsoft.AppCenter.Utils.Constants.AppCenterFilesDirectoryPath = Environment.CurrentDirectory;
+            Microsoft.AppCenter.Utils.Constants.AppCenterDatabasePath = DatabasePath;
+            try
+            {
+                adapter.Initialize(DatabasePath);
+            }
+            catch
+            {
+                // Handle exception, database is not created with Mock.
+            }
+            Assert.IsTrue(System.IO.File.Exists(DatabasePath));
```

Comment:
To avoid other test cases interfering, it would be ideal to check `DatabasePath` doesn't exist before `adapter.Initialize` call.